helm-charts/konk/scripts/provision.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -xe
 
+secret_not_found () {
+  if [ "$(kubectl -n $NAMESPACE get secret $1 --ignore-not-found 2>&1 )" = "" ]
+  then
+    return 0
+  else
+    return 1
+  fi
+}
+
 # load existing CA
 if kubectl -n $NAMESPACE get secret $FULLNAME-ca
```

Comment:
Use it here too. If `kubectl` returns an error, this should retry or abort rather than skipping loading the CA cert.